In file included from Source/GmmLib/ULT/GmmAuxTableULT.cpp:25:
In file included from Source/GmmLib/ULT/./GmmAuxTableULT.h:33:
/usr/include/malloc.h:3:2: error: "<malloc.h> has been replaced by <stdlib.h>"
#error "<malloc.h> has been replaced by <stdlib.h>"
 ^

--- Source/GmmLib/ULT/GmmAuxTableULT.h.orig	2019-09-16 10:30:27 UTC
+++ Source/GmmLib/ULT/GmmAuxTableULT.h
@@ -30,7 +30,6 @@ OTHER DEALINGS IN THE SOFTWARE.
 
 #include "GmmGen10ResourceULT.h"
 #include <stdlib.h>
-#include <malloc.h>
 
 #ifndef ALIGN
 #define ALIGN(v, a) (((v) + ((a)-1)) & ~((a)-1))
